welcomebot.py

```diff
@@ -187,24 +187,31 @@ def perform_image_search():
     elif message.content.startswith("//search"):
         print "Is search request"
         if len(message.content.split()) == 1:
-            message.message.reply("Can't search, No terms given")
+            message.message.reply("Can't search, no terms given")
         else:
             def perform_google_search():
                 search_term = "+".join(message.content.split()[1:])
-                search_results = google_search.google_search(search_term)
-                print search_results
-                if search_results is False:
-                    print "No Results"
+                try:
+                    results, moreResultsUrl = google_search.google_search(search_term)
+                    print results
+                except TypeError:
```

Comment:
If you cannot extract enough values from a return value, Python throws a ValueError, not a TypeError.